invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -16,14 +16,17 @@
 ## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
 import collections
+import csv
+import os.path
 
 from invenio.ext.sqlalchemy import db
 from flask import current_app
 from datetime import date
 
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
wouldn't be better to use pkg_resources for that? see http://stackoverflow.com/questions/779495/python-access-data-in-package-subdirectory